android/src/main/java/com/brentvatne/react/ReactVideoView.java

```diff
@@ -457,6 +459,42 @@ public void setControls(boolean controls) {
         this.mUseNativeControls = controls;
     }
 
+    public boolean isPreventScreenFromDimmingFlagOn() {
+        int flags = mThemedReactContext.getCurrentActivity().getWindow().getAttributes().flags;
+
+        if ((flags & WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON) == 0) {
```

Comment:
If we need to keep this check for performance or other reasons, it can be written as:
`return flags & WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON == 0`